### PR TITLE
ScreenToGif 2.36

### PR DESCRIPTION
**Pull request by NickeManarin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/45988)

### CODE REVIEW

File: manifests/n/NickeManarin/ScreenToGif/2.36.0/NickeManarin.ScreenToGif.locale.pt-BR.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: NickeManarin.ScreenToGif
+PackageVersion: 2.36.0
```
Comment (OfficialEsco) [inline manifests/n/NickeManarin/ScreenToGif/2.36.0/NickeManarin.ScreenToGif.locale.pt-BR.yaml:5]:
```suggestion
PackageVersion: 2.36
```

File: manifests/n/NickeManarin/ScreenToGif/2.36.0/NickeManarin.ScreenToGif.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: NickeManarin.ScreenToGif
+PackageVersion: 2.36.0
```
Comment (OfficialEsco) [inline manifests/n/NickeManarin/ScreenToGif/2.36.0/NickeManarin.ScreenToGif.yaml:5]:
```suggestion
PackageVersion: 2.36
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
This is probably something you can and should fix on your side since a consistent 3 digit version number looks better

**Comment by NickeManarin** [comment]
@OfficialEsco I was following the pattern of other packages, such as this one https://github.com/microsoft/winget-pkgs/pull/45894/files

**Comment by OfficialEsco** [comment]
> @OfficialEsco I was following the pattern of other packages, such as this one https://github.com/microsoft/winget-pkgs/pull/45894/files
>
>To have the 3 number groups.
>X.X.0

I'm sorry my message might have been a bit vague and written wrongly, this current version writes `2.36` as its DisplayVersion in the registry, i was trying to advice you to change it to `2.36.0` as you used in the PR because it looks better

I meant 3 number groups instead of 3 digits 🤦‍♂️

**Comment by OfficialEsco** [comment]
Yep 👍

**Comment by NickeManarin** [comment]
Ohh, ok.
I prefer something like that which looks prettier to me. :)
I usually follow this format, # meaning that it only appears if it's greater than 0.
 
```
0.0.#.#
3.0       == 3.0.0.0
2.36      == 2.36.0.0
2.36.1    == 2.36.1.0
2.36.2.15 == 2.36.2.15
```

**Comment by OfficialEsco** [comment]
Yeah that is understandable, but it can get messy when sorting and stuff when it changes between 2 and 3 groupings, Microsoft prefers [Semantic Versioning](https://semver.org/) because it covers Major, Minor/Feature and Patch/Hotfix, IMO i rarely see Publishers use the 4th group for anything special

**Comment by NickeManarin** [comment]
Correcting myself, I'm using WixSharp to create the installer and it simply accepts a .NET Version object, with no easy to spot way to control how it sets the version number internally. 

I'll have to check. If I find something, I'll use it for v2.38 or 3.0.0. :)

**Review by OfficialEsco — APPROVED** [review]
Alrighty, ill accept this since winget ignores the .0’s

**Pull request merged**